ucan-key-support/Cargo.toml

```diff
@@ -20,26 +20,26 @@ version = "0.7.0-alpha.1"
 default = []
 
 [dependencies]
-anyhow = "1.0.52"
-async-trait = "0.1.52"
+anyhow = "1.0"
```

Comment:
Why not `^1`?

Reply:
most libraries tend to avoid the default strategy w/ `^`, as they are not really needed: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements.

Reply:
> Caret requirements are an alternative syntax for the default strategy, ^1.2.3 is exactly equivalent to 1.2.3.

Ah, okay. TIL.

Reply:
Rust has whole lot of SemVer compatibility footguns to be aware of - this is why it is common to restrict major.minor

Reply:
@pinkforest yep. yeah, that's why I'm typically major.minor.*